apps/language_server/lib/language_server/experimental/provider/code_action/replace_with_underscore.ex

```diff
@@ -5,28 +5,26 @@ defmodule ElixirLS.LanguageServer.Experimental.Provider.CodeAction.ReplaceWithUn
 
   alias ElixirLS.LanguageServer.Experimental.CodeMod
   alias ElixirLS.LanguageServer.Experimental.CodeMod.Ast
-  alias ElixirLS.LanguageServer.Experimental.Protocol.Requests.CodeAction
   alias ElixirLS.LanguageServer.Experimental.Protocol.Types.CodeAction, as: CodeActionResult
   alias ElixirLS.LanguageServer.Experimental.Protocol.Types.Diagnostic
   alias ElixirLS.LanguageServer.Experimental.Protocol.Types.TextEdit
   alias ElixirLS.LanguageServer.Experimental.Protocol.Types.Workspace
   alias ElixirLS.LanguageServer.Experimental.SourceFile
 
-  @spec apply(CodeAction.t()) :: [CodeActionResult.t()]
-  def apply(%CodeAction{} = code_action) do
-    source_file = code_action.source_file
-    diagnostics = get_in(code_action, [:context, :diagnostics]) || []
+  @pattern ~r/variable "([^"]+)" is unused/
```

Comment:
nit: Pattern is a much less descriptive name than `variable_re`, but we should not be exposing this anyways.